modules/table/src/main/java/org/apache/ignite/internal/table/distributed/StorageUpdateHandler.java

```diff
@@ -81,8 +109,32 @@ public void handleUpdate(
             TablePartitionId commitPartitionId,
             @Nullable ByteBuffer rowBuffer,
             @Nullable Consumer<RowId> onReplication
+    ) {
+        handleUpdate(txId, rowUuid, commitPartitionId, rowBuffer, onReplication, null);
+    }
+
+    /**
+     * Handles single update.
+     *
+     * @param txId Transaction id.
+     * @param rowUuid Row UUID.
+     * @param commitPartitionId Commit partition id.
+     * @param rowBuffer Row buffer.
+     * @param onReplication Callback on replication.
+     * @param lowWatermark GC low watermark.
```

Comment:
What does it mean that `null` as passed as a watermark?

Reply:
That’s for us to find out when LWM will be implemented :)